### ISSUE TITLE
.gitignore ignores ./compose/local/* files

### ISSUE THREAD

**Issue by markliederbach** [pydanny/cookiecutter-django#1321]
**Note: for support questions, please use the `cookiecutter-django` tag on stackoverflow**. This repository's issues are reserved for feature requests and bug reports. If you need quick professional paid support for your project, contact support@cookiecutter.io.

* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

* **Do you want to request a *feature* or report a *bug*?** Bug

* **What is the current behavior?** When committing a newly generated project to Github, the .gitignore filters out the `local` directory in the `compose` folder. This means that the generated `local.yml` file returns an error if you tried to run it.

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** 

1. Generate a project:
```
root@b6296e448afe:/# cookiecutter https://github.com/pydanny/cookiecutter-django
project_name [Project Name]: Blog Backend
project_slug [blog_backend]:
author_name [Daniel Roy Greenfeld]: Mark Liederbach
email [you@example.com]: contact@example.com
description [A short description of the project.]:
domain_name [example.com]: example.com
version [0.1.0]: 0.0.1
timezone [UTC]:
use_whitenoise [y]:
use_celery [n]:
use_mailhog [n]: y
use_sentry_for_error_reporting [y]:
use_opbeat [n]:
use_pycharm [n]: y
windows [n]:
use_docker [n]: y
use_heroku [n]:
use_elasticbeanstalk_experimental [n]:
use_compressor [n]:
Select postgresql_version:
1 - 9.6
2 - 9.5
3 - 9.4
4 - 9.3
5 - 9.2
Choose from 1, 2, 3, 4, 5 [1]: 1
Select js_task_runner:
1 - Gulp
2 - Grunt
3 - None
Choose from 1, 2, 3 [1]: 3
custom_bootstrap_compilation [n]:
Select open_source_license:
1 - MIT
2 - BSD
3 - GPLv3
4 - Apache Software License 2.0
5 - Not open source
Choose from 1, 2, 3, 4, 5 [1]: 1
```
Note that this **does** create a `local` directory at `./compose/local`

2. Push to Github. The folder is ignored.

3. Attempt to run docker-compose with the local.yml file. It won't work.

* **What is the expected behavior?** The local directory should not be ignored.

* **What is the motivation / use case for changing the behavior?** Without it, I either need to hunt down the rule ignoring it to delete it, or I must manually copy the local directory's contents to a new workstation.

* **Please tell us about your environment:** Developing in Pycharm for Windows, but building and running the app using linux docker.

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by markliederbach** [comment]
A current workaround is to add the following to the bottom of the `.gitignore`
```
!/compose/local/
```

**Comment by sfdye** [comment]
This is caused by PR #1317, conflicting with:
https://github.com/pydanny/cookiecutter-django/blob/a42df4f1b2e309b4b7f2aa4ed2587bf6402b5acd/%7B%7Bcookiecutter.project_slug%7D%7D/.gitignore#L353

**Comment by webyneter** [comment]
Closed by #1322.
